tools/importer-rest-api-specs/components/parser/constants_test.go

```diff
@@ -284,6 +284,96 @@ func TestParseConstantsIntegersInlinedAsIntsWithDisplayName(t *testing.T) {
 	validateParsedSwaggerResultMatches(t, expected, actual)
 }
 
+func TestParseConstantsMultipleTypeEnums(t *testing.T) {
+	actual, err := ParseSwaggerFileForTesting(t, "constants_multiple_type_enums.json")
+	if err != nil {
+		t.Fatalf("parsing: %+v", err)
+	}
+
+	expected := importerModels.AzureApiDefinition{
+		ServiceName: "Example",
+		ApiVersion:  "2020-01-01",
+		Resources: map[string]importerModels.AzureApiResource{
+			"Animal": {
```

Comment:
(as below) can we combine these into the same API Resource, so that we can ensure we're testing that we get unique names back within the same API Resource?